src/SdkGenerator/SdkGenerator.csproj

```diff
@@ -1,74 +1,85 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <OutputType>Exe</OutputType>
-        <TargetFramework>net7.0</TargetFramework>
-        <PackAsTool>true</PackAsTool>
-        <ToolCommandName>SdkGenerator</ToolCommandName>
-        <PackageOutputPath>../../</PackageOutputPath>
-    </PropertyGroup>
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>SdkGenerator</ToolCommandName>
+    <PackageReadmeFile>docs/README.md</PackageReadmeFile>
+    <Description>A command line dotnet tool to retrieve swagger (OpenAPI) files and generate
+      easy-to-use markdown documentation or friendly software development kit libraries. Can be
+      automated using Github Workflows.</Description>
+    <PackageTags>SDK generator swagger openapi swashbuckle</PackageTags>
+    <Copyright>Copyright 2021 - 2023</Copyright>
+    <PackageReleaseNotes># 1.1.6
+      August 21, 2023
+
+      * Fixed capitalization of DotnetToolSettings.xml
+    </PackageReleaseNotes>
+    <PackageIcon>docs/icons-puzzle.png</PackageIcon>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Version>1.1.6</Version>
+    <Authors>Ted Spence</Authors>
+    <!-- Project Url is filled in by sourcelink in the .NET 8 SDK, but you can add it explicitly via
+    package -->
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+  </PropertyGroup>
 
-    <ItemGroup>
-        <PackageReference Include="CommandLineParser" Version="2.9.1" />
-        <PackageReference Include="HtmlAgilityPack" Version="1.11.50" />
-        <PackageReference Include="JsonDiffPatch.Net" Version="2.3.0" />
-        <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
-        <PackageReference Include="RestSharp" Version="110.2.0" />
-        <PackageReference Include="Scriban" Version="5.7.0" />
-        <PackageReference Include="Semver" Version="2.3.0" />
-    </ItemGroup>
+  <ItemGroup>
```

Comment:
Sorry about the formatting - my editor auto-formatted the file

Reply:
No worries!